scripts.js

```diff
@@ -11,7 +11,7 @@ document.addEventListener('keydown', (e) => {
 		playerOne.style.left = distance1 + 'px';
 	}
 	else if (distance1 === 1100) {
-		alert('Player One is the WINNER!');
+		alert('Green car is the WINNER!');
```

Comment:
In the future, I would recommend some other indicator besides alerts that the player has won (maybe just load some text into the DOM that says "Blue won" or something like that). It isn't wrong to use alerts per se, but using something else can take your app from hacky to classy with very little effort! :)